pack-common/src/main/java/org/apache/servicecomb/pack/common/Environment.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.pack.common;
+
+public class Environment {
+  private static Environment instance;
+  private static final int PAYLOADS_MAX_LENGTH = 10240;
+  private int payloadsMaxLength = 0;
+
+  public Environment() {
+    if (payloadsMaxLength == 0) {
+      String val = System.getenv("PAYLOADS_MAX_LENGTH");
+      if (val == null || val.trim().length() == 0) {
+        payloadsMaxLength = PAYLOADS_MAX_LENGTH;
+      } else {
+        try {
+          payloadsMaxLength = Integer.parseInt(val);
+        } catch (NumberFormatException ex) {
```

Comment:
We need to add a log to specify the error and fallback action.

Reply:
done